### PR TITLE
Trait Implementation for QueryBuilder and Removal of some deprecated code blocks

### PR DESCRIPTION
**Pull request by hojalatheef**

### CODE REVIEW

File: Classes/Trait/QueryBuilderTrait.php [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+declare(strict_types=1);
+
+/*
+ * This file is part of the package jweiland/service-bw2.
+ *
+ * For the full copyright and license information, please read the
+ * LICENSE file that was distributed with this source code.
+ */
+
+namespace JWeiland\ServiceBw2\Trait;
```
Comment (sfroemkenjw) [inline Classes/Trait/QueryBuilderTrait.php:12]: Traits

Trait is a php reserved keyword

File: Classes/ViewHelpers/OpeningHoursViewHelper.php [hunk 2]
```diff
@@ -125,12 +125,12 @@ protected static function getStructuredOpeningHoursHTML(array $structuredOpening
...
 
             // show note if set
-            if (isset($structuredOpeningHours['hinweisText']) && $structuredOpeningHours['hinweisText']) {
+            if ($structuredOpeningHours['hinweisText'] ?? false) {
                 $html[] = '<dd class="structured-opening-hours">' . $structuredOpeningHours['hinweisText'] . '</dd>';
             }
             //todo: use StandaloneView to use a fluid template instead of manual rendering.
             foreach (self::DAYS as $dayInGerman) {
-                $afternoon = $afternoonOpeningHours[$dayInGerman] ?? false;
+                $afternoon = !empty($afternoonOpeningHours[$dayInGerman]);
```
Comment (sfroemkenjw) [inline Classes/ViewHelpers/OpeningHoursViewHelper.php:133]: ihhh...empty ;-)

File: Configuration/FlexForms/ServiceBw2FlexFormConfiguration.xml [hunk 3]
```diff
@@ -33,10 +29,9 @@
                             <displayCond>FIELD:parentRec.CType:=:servicebw2_organizationalunitsshow</displayCond>
                             <config>
                                 <type>group</type>
-                                <internal_type>db</internal_type>
                                 <allowed>pages</allowed>
                                 <size>1</size>
-                            </config>
+                            </config>y
```
Comment (sfroemkenjw) [inline Configuration/FlexForms/ServiceBw2FlexFormConfiguration.xml:34]: y

**Review by sfroemkenjw — CHANGES_REQUESTED** [review]

File: Classes/Trait/QueryBuilderTrait.php (hunk 1 above)
Comment (hojalatheef) [reply Classes/Trait/QueryBuilderTrait.php]: Aww!! How I missed that ;)

File: Classes/ViewHelpers/OpeningHoursViewHelper.php (hunk 2 above)
Comment (hojalatheef) [reply Classes/ViewHelpers/OpeningHoursViewHelper.php]: Hah! ofcourse I know a person who hates empty now ;)

File: Configuration/FlexForms/ServiceBw2FlexFormConfiguration.xml (hunk 3 above)
Comment (hojalatheef) [reply Configuration/FlexForms/ServiceBw2FlexFormConfiguration.xml]: this is rubbish!! and embarassing because of keyboard errors.

**Pull request merged**